src/ai/backend/manager/models/gql_models/group.py

```diff
@@ -116,6 +94,28 @@ class Meta:
         UserConnection,
     )
 
+    queryfilter_fieldspec: Mapping[str, FieldSpecItem] = {
+        "id": ("id", None),
+        "row_id": ("id", None),
```

Comment:
This is introduced by https://app.graphite.dev/github/pr/lablup/backend.ai/3525/feat-BA-595-Add-row_id-field-to-UserNode-GQL-schema. But we didn't finish the discussion. Let's remove `row_id`.